paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/ui/shared/Receipt.kt

```diff
@@ -153,29 +146,31 @@ fun ProductRow(
         Row(
             verticalAlignment = Alignment.CenterVertically,
         ) {
-            IconButton(
-                onClick = {
-                    val currentQuantity = quantity ?: 0
-                    onQuantityChanged(currentQuantity - 1)
-                },
-                enabled = !isProcessing && isEditable && (quantity ?: 0) > 0,
-            ) {
-                Icon(imageVector = Icons.Default.Remove, contentDescription = null)
+            if (isEditable) {
+                IconButton(
+                    onClick = { onQuantityChanged(quantity - 1) },
+                    enabled = !isProcessing && quantity > 0,
+                ) {
+                    Icon(imageVector = Icons.Default.Remove, contentDescription = null)
+                }
             }
 
             Text(
-                text = quantity?.toString().orEmpty(),
+                text = if (isEditable) {
+                    quantity.toString()
+                } else {
+                    "${quantity}x"
+                },
                 fontSize = MAIN_FONT_SIZE,
             )
 
-            IconButton(
-                onClick = {
-                    val currentQuantity = quantity ?: 0
-                    onQuantityChanged(currentQuantity + 1)
-                },
-                enabled = !isProcessing && isEditable,
-            ) {
-                Icon(imageVector = Icons.Default.Add, contentDescription = null)
+            if (isEditable) {
+                IconButton(
+                    onClick = { onQuantityChanged(quantity + 1) },
+                    enabled = !isProcessing,
+                ) {
+                    Icon(imageVector = Icons.Default.Add, contentDescription = null)
```

Comment:
This is from previous, but this should really have a contentDescription!

Reply:
I can add some as a follow-up!